src/static/templateUrl.test.ts

```diff
@@ -6,6 +6,10 @@
  */
 import TemplateURL from './templateUrl';
 
+// TODO: add unit tests for templateURL
```

Comment:
May be we can add tests in the PR and rest can be a separate ticket if needed

Reply:
Added unit tests!